neuralprophet/time_net.py

```diff
@@ -559,7 +649,19 @@ def forward(self, inputs):
                 )
 
         trend = self.trend(t=inputs["time"])
-        out = trend + additive_components + trend.detach() * multiplicative_components
+
+        # 0 is the median quantile index
+        # all multiplicative components are multiplied by the median quantile trend
```

Comment:
Multiplying by the median quantile trend is also from Hansika's code. Since I'm not sure what is the benefit of doing that instead of multiplying by respective quantiles (as you said), I have reverted it back to the original:

`trend + additive_components + trend.detach() * multiplicative_components
`
Nevertheless, I still kept that code (in commented form) just in case.